test/_features/example_app/database.feature

```diff
@@ -0,0 +1,48 @@
+@feature @save_and_return
```

Comment:
My suggestion might be to name this file save_and_return.feature

Reply:
changed the file names, thanks